content/docs/command-reference/add.md

```diff
@@ -243,3 +246,44 @@ In this case, a DVC-file is generated for each file in the `pics/` directory
 tree. No top-level DVC-file is generated, which is typically less convenient.
 For example, we cannot use the directory structure as one unit with `dvc run` or
 other commands.
+
+## Example: Dvcignore
+
+Let's take an example to illustrate how `.dvcignore` interacts with `dvc add`.
+
+```dvc
+$ mkdir dir
+$ echo file_one > dir/file1
+$ echo file_two > dir/file2
+```
+
+Now add `file1` to `.dvcignore` and track the entire `Dir` directory with
+`dvc add`.
+
+```dvc
+$ echo Dir/file1 > .dvcignore
+$ dvc add Dir
+```
```

Comment:
I had changed all the `Dir` to `dir` but now somehow there's a mix. Maybe a bad merge, did you have to solve conflicts at some point? Anyway, please use `dir` everywhere.

Reply:
> I had changed all the Dir to dir but now somehow there's a mix.
I guess

Reply:
Thanks for using `dir` now.

No, Git is smart enough not to let you overwrite other people's changes unless you solve conflicts that way specifically and/or force-push.

Anyway, it's fixed now.